src/lib/server/embeddingEndpoints/transformersjs/embeddingEndpoints.ts

```diff
@@ -1,6 +1,6 @@
 import { z } from "zod";
 import type { EmbeddingEndpoint } from "../embeddingEndpoints";
-import type { Tensor, Pipeline } from "@xenova/transformers";
+import type { Tensor, FeatureExtractionPipeline } from "@xenova/transformers";
```

Comment:
double checking: this change is unrelated to tokenization (this PR) right? I think we can still keep it

Reply:
yes I was having type related issues with the update to transformers js so unrelated but needed